### PR TITLE
Create bookmark page

### PR DESCRIPTION
**Pull request by Seborider**
[Deplyoment](https://review-page-bookmark-witlbxytd.herokuapp.com/bookmark)

### CODE REVIEW

File: src/app/pages/Bookmarks/Bookmarks.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import styles from './Bookmarks.module.css';
+import Header from '../../components/Header/Header';
+import Card, { CardProps } from '../../components/Card/Card';
+import Navigation from '../../components/Navigation/Navigation';
+
+// type MovieInfo = {
+//   movieInfo: string;
+// };
```
Comment (marwinburesch) [inline src/app/pages/Bookmarks/Bookmarks.tsx:9]: 😵

File: src/app/pages/Bookmarks/Bookmarks.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      children: '???',
+    },
+    {
+      layout: 'detail',
+      title: 'Lethal Weapon',
+      rating: 5,
+      genres: [''],
+      image: 'string | null',
+      isBookmarked: true,
+      onBookmarkClick: () => console.log(''),
```
Comment (marwinburesch) [inline src/app/pages/Bookmarks/Bookmarks.tsx:40]: I would've expected this function to be declared on the page itself and handed to every component inside the `.map` in line 54

File: src/app/pages/Bookmarks/Bookmarks.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <div className={styles.container}>
+      <section className={styles.header}>
+        <Header isHighlighted withBackButton>
+          Bookmarks
+        </Header>
+      </section>
+      <main className={styles.cardWrapper}>
+        {movieInfo.map((movie) => (
+          <Card key={movie.title} {...movie}></Card>
```
Comment (marwinburesch) [inline src/app/pages/Bookmarks/Bookmarks.tsx:55]: You can write this as a void element `<Card key={movie.title} {...movie} />`

File: src/app/pages/Bookmarks/Bookmarks.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        ))}
+      </main>
+      <section className={styles.navigation}>
+        <Navigation activeLink="bookmark"></Navigation>
+      </section>
```
Comment (marwinburesch) [inline src/app/pages/Bookmarks/Bookmarks.tsx:60]: Sizing should be possible without setting any styles for this component. Checkout [this PR](https://github.com/marwinburesch/moviescout/pull/23/files)

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Awesome work, cheers! ☔

File: src/app/pages/Bookmarks/Bookmarks.tsx (hunk 1 above)
Comment (Seborider) [reply src/app/pages/Bookmarks/Bookmarks.tsx:9]: Resolved

File: src/app/pages/Bookmarks/Bookmarks.tsx (hunk 2 above)
Comment (Seborider) [reply src/app/pages/Bookmarks/Bookmarks.tsx:40]: Resolved

File: src/app/pages/Bookmarks/Bookmarks.tsx (hunk 3 above)
Comment (Seborider) [reply src/app/pages/Bookmarks/Bookmarks.tsx:55]: Resolved

File: src/app/pages/Bookmarks/Bookmarks.tsx (hunk 4 above)
Comment (Seborider) [reply src/app/pages/Bookmarks/Bookmarks.tsx:60]: Resolved

File: src/app/pages/Bookmarks/Bookmarks.module.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  z-index: 100;
+  background-color: var(--bg-primary);
+}
+
+.cardWrapper {
+  display: grid;
+  padding: 1em;
+  gap: 1em;
+  overflow: scroll;
+}
```
Comment (marwinburesch) [inline src/app/pages/Bookmarks/Bookmarks.module.css:26]: Maybe some responsiveness, with a `repeat(auto-fit, minmax())` type grid?
https://github.com/marwinburesch/moviescout/pull/26#pullrequestreview-741660803

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Thank you, almost there!

File: src/app/pages/Bookmarks/Bookmarks.module.css (hunk 5 above)
Comment (Seborider) [reply src/app/pages/Bookmarks/Bookmarks.module.css]: Resolved in commit [685c03a](https://github.com/marwinburesch/moviescout/pull/27/commits/685c03a89f1c642ec291f67248383e8374ac2fc2)

**Review by phlgr — APPROVED** [review]

**Review by marwinburesch — APPROVED** [review]

**Pull request merged**